helpers/code.go

```diff
@@ -7,32 +7,43 @@ import (
 	"strconv"
 )
 
-var regexpCodeGrabber = regexp.MustCompile(`^26bda8cd-ad49-4e4b-8bb3-2f19e197b3bd\[(\d+)\]$`)
+var regexpCodeGrabber = regexp.MustCompile(`^26bda8cd-ad49-4e4b-8bb3-2f19e197b3bd\[(\d+)\]\[(.*?)\]$`)
```

Comment:
It doesn't fail but does not work as expected. The exit code isn't not used and the code sequence is displayed in the output.